### ISSUE TITLE
Provide pypi package

### ISSUE THREAD

**Issue by ghost** [MTG/acousticbrainz-client#53]
`setup.py` already exists, so it would be great if you could take the additional step of pushing it to pypi (or if that's not possible to provide a docker image with instructions)

**Comment by alastair** [comment]
good idea, thanks. This year we're planning on making an update to the software that computes features, and so we'll definitely add the client to pypi then.
